main.tf

```diff
@@ -555,7 +555,8 @@ data "aws_iam_policy_document" "ecs_task_access_secrets" {
       aws_ssm_parameter.webhook.*.arn,
       aws_ssm_parameter.atlantis_github_user_token.*.arn,
       aws_ssm_parameter.atlantis_gitlab_user_token.*.arn,
-      aws_ssm_parameter.atlantis_bitbucket_user_token.*.arn
+      aws_ssm_parameter.atlantis_bitbucket_user_token.*.arn,
+      var.repository_credentials != null && can(var.repository_credentials["credentialsParameter"]) ? [var.repository_credentials["credentialsParameter"]] : []
```

Comment:
Based on [terraform docs](https://www.terraform.io/language/functions/can) there's a preference to use `try` instead of `can` for this kind of try/catch logic. You can also see how try is used elsewhere in this module. 

```suggestion
      try(var.repository_credentials["credentialsParameter"], []) 
```

Reply:
yes! and since this might come from another resource, it would be better if we use `lookup()` here
```suggestion
      lookup(var.repository_credentials["credentialsParameter"], [])
```